include/boost/geometry/algorithms/detail/calculate_sum.hpp

```diff
@@ -30,7 +34,7 @@ class calculate_polygon_sum
     template <typename ReturnType, typename Policy, typename Rings, typename Strategy>
     static inline ReturnType sum_interior_rings(Rings const& rings, Strategy const& strategy)
     {
-        ReturnType sum = ReturnType();
+        ReturnType sum = ReturnType(0);
```

Comment:
Thanks Adam for looking up the message of Bruno. 
And thanks for grepping.
OK, I agree with 0 initialization.